bin/git-info

```diff
@@ -21,27 +36,39 @@ remote_urls() {
 }
 
 echon() {
-    echo "$@"
-    echo
+  echo "$@"
+  echo
 }
 
-# Show info similar to svn
+GREEN="$(tput setaf 2)"
+NORMAL="$(tput sgr0)"
+if [ "$1" = "--color" ] || [ "$2" = "--color" ] || \
+     [ "$1" = "-c" ] || [ "$2" = "-c" ] ; then
+  COLOR_TITLE="$GREEN"
```

Comment:
Better to mention that `-c` only affects the title in the documentation.

Reply:
Fixed